### DISCUSSION TITLE
How to install/run Fedora 40 with Quickemu

### DISCUSSION THREAD

**Discussion by lucamosca1** [quickemu-project/quickemu#1252] [Quickemu] [answered]
Environment: quickemu installed on Ubuntu 22.04

1 - Is it the correct iso?
iso downloaded is Fedora-Workstation-Live-osb-40-1.14.x86_64.iso, apart the osb variant, when running I see the pre-release alert in red like the beta version.

2 - Empty window on gnome
When running fedora, content of window are almost empty, making the installation process (and user setup) very difficult to complete, see attachment
![Schermata del 2024-05-22 15-07-52](https://github.com/quickemu-project/quickemu/assets/5502068/614c94a7-aa80-4a43-8d8d-2f779f2c77e6)

**Comment by flexiondotorg** [comment]
Please supply all the information requested via the issue template.

**Comment by lj3954** [comment, marked as answer]
I can replicate this. However, I believe this to be an issue with the media distributed by Fedora, not with quickemu. It does not impact the usability of the distribution in any way, you can install it normally using the anaconda installer linked to in the dash and everything should be fine post-install.

**Comment by lucamosca1** [comment]
Issue IMHO more widespread than expected, reproducible also on Ubuntu 24.04
![Schermata del 2024-05-28 12-23-50](https://github.com/quickemu-project/quickemu/assets/5502068/17eabc6a-21bc-49e3-a8ae-95fdc74ac72e)

Difficult to understand if it's a gnome bug or quickemu bug

**Comment by lucamosca1** [comment]
Opened bug also gnome side
https://gitlab.gnome.org/GNOME/gnome-initial-setup/-/issues/220
